### DISCUSSION TITLE
/v2/zh/guide/get-started/install.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#192] [Announcements]
# /v2/zh/guide/get-started/install.html

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/get-started/install.html

**Comment by wlsh123** [comment]
npm init vuepress-theme-hope@next 命令没有任何反应是啥情况

**Reply by 000luna000** [reply]
test

**Reply by HaomingCai** [reply]
我也一样

**Reply by HaomingCai** [reply]
我好了

**Comment by wind-liang** [comment]
本文档配置 地址 404 了

**Comment by itwanger** [comment]
如何直接升级呢？
